.vscode/launch.json

```diff
@@ -0,0 +1,45 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "lldb",
```

Comment:
this is a configuration file for debugging rust with LLDB in VScode (with the lldb-vscode adapter extension)